packages/mds-web-sockets/package.json

```diff
@@ -39,7 +39,7 @@
     "start": "yarn watch server",
     "test": "yarn test:eslint && yarn test:unit",
     "test:eslint": "eslint --ignore-path ../../.gitignore '**/*.ts'",
-    "test:unit": "DOTENV_CONFIG_PATH=../../.env nyc --lines 50 ts-mocha --project ../../tsconfig.json --exit",
+    "test:unit": "DOTENV_CONFIG_PATH=../../.env jest --forceExit",
```

Comment:
`--forceExit` 😱

Reply:
Yeah... I need to add a ProcessManager wrapper to the WS service, right now there's no good way to safe-close it. Wanna save that for another PR though (but it's certainly on the bucket list of things I want to enhance for WS in Q4)

Reply:
Resolved in https://github.com/lacuna-tech/mds-core/pull/487